README.md

```diff
@@ -133,7 +133,12 @@ At the top of this hierarchy is an abstract class **PreMetric**, which is define
 
 	d(x, z) <= d(x, y) + d(y, z)  for all x, y, z
 
-This type system has practical significance. For example, when computing pairwise distances between a set of vectors, you may only perform computation for half of the pairs, and derive the values immediately for the remaining halve by leveraging the symmetry of *semi-metrics*.
+This type system has practical significance. For example, when computing pairwise distances between a set of vectors, you may only
+perform computation for half of the pairs, and derive the values immediately for the remaining halve by leveraging the symmetry of
+*semi-metrics*. This is why the type definitions for `SemiMetric` and
+`Metric` do not completely follow the mathematical definition of these
+structures which include positive definiteness: `x != y` does not imply
+that `d(x, y) != 0` in general, as this does not change computations.
```

Comment:
Although I can read/parse it, this sentence is a little bit long and hard to read for me... I think it isn't so obvious which `which` in "which include positive definiteness" and `this` in "as this does not change computations" refer to.

Reply:
Thanks for the swift review, will try to rewrite more clearly.